controllers/cluster/rbac.go

```diff
@@ -42,7 +43,7 @@ func createClusterRBAC(obj pipeline.Object, data *pipeline.Context) pipeline.Res
 	}
 	for _, item := range []client.Object{serviceAccount, role, roleBinding} {
 		if err := data.Client.Create(context.TODO(), item); err != nil && !errors.IsAlreadyExists(err) {
-			return pipeline.Result{Err: err}
+			return pipeline.Result{Err: fmt.Errorf("creating %s/%s: %w", item.GetObjectKind().GroupVersionKind().Kind, item.GetName(), err)}
```

Comment:
```suggestion
			return pipeline.Result{Err: fmt.Errorf("creating %s '%s': %w", item.GetObjectKind().GroupVersionKind().Kind, item.GetName(), err)}
```

Reply:
Actually, the GVK on newly created structs might empty if not explicitly set, the Client figures this out almost via reflection. So the `item.GetObjectKind().GroupVersionKind().Kind` might be empty at this time.
I'm not quite sure though. Did you try it?